pkg/state/state.go

```diff
@@ -809,6 +812,8 @@ func (st *HelmState) DeleteReleasesForSync(affectedReleases *AffectedReleases, h
 					} else {
 						affectedReleases.Deleted = append(affectedReleases.Deleted, release)
 					}
+					elapsed := time.Since(start)
+					release.Duration = elapsed
```

Comment:
why not `release.Duration = time.Since(start)`

Reply:
My mistake, changed it to what you suggested